backend/auth/main.py

```diff
@@ -85,10 +147,10 @@ async def get_current_story(
         if not (user and user.story):  # no story nor user match the token data
             raise HTTPException(status_code=404, detail="Story not found")
         else:  # there's a user with a story
-            return schemas.Story.from_module(user.story)
+            return story_schemas.Story.from_orm(user.story)
```

Comment:
as you pointed out in this [issue-comment](https://github.com/instedd/oasis/pull/41#discussion_r431821487), shouldn't we return a model-story here instead of a schema-story?

Reply:
That was aimed to crud methods, to keep them consistent... but it might be worth doing here, since we're at it. I'll remove it here, but I'll have to manually add it in the endpoints that weren't mapping automatically the model (see your comments on `routers.stories` for more on that).